_docs/releasing/index.md

```diff
@@ -67,10 +67,9 @@ Use [new-project](https://github.com/perploug/new-project)(TODO: move to zalando
 Besides the rules of play, there is also a set of best practices which we highly recommend you implement.
 
 1.  Have a code of conduct and enforce it to create a safe environment for collaboration
-2.  Have a clear vision and scope of your project
-3.  Set clear expectations for responses - let users know if your time is limited
-4.  Ask for help and be open to what kind of contributions would help your project
-5.  Be mindful of your documentation
+2.  Set clear expectations for responses - let users know if your time is limited
```

Comment:
Don't think this clause should be removed - it might not be a requirement for incubation - but it is a best practices and a project should think of this as early as possible